src/test/regress/expected/resync_metadata_with_sequences.out

```diff
@@ -141,6 +141,8 @@ INSERT INTO sensors_news VALUES (DEFAULT, DEFAULT, '2021-01-01') RETURNING *;
 
 \c - - - :master_port
 SELECT 1 FROM citus_activate_node('localhost', :worker_1_port);
```

Comment:
should we increase the log level here as well to avoid future tests changing the message?